src/engine/ruleset/integrations/system/decoders/system-auth.yml

```diff
@@ -0,0 +1,116 @@
+name: decoder/system-auth/0
+
+sources:
+  - decoder/syslog/0
+
+metadata:
+  module: system-auth
+  title: system-auth logs
+  description: Decoder for system athenticated action logs.
+  compatibility: This integration was tested with logs from OS like Ubuntu 12.04, Centos 7, and macOS Sierra.
+  versions:
+    - "any"
+  author:
+    name: Wazuh Inc.
+    email: info@wazuh.com
+    date: 2023-05-15
+  references:
+    - https://www.loggly.com/ultimate-guide/linux-logging-basics/
+
+definitions:
+  isAuthProcess: process.name=="sshd" OR process.name=="sudo" OR process.name=="groupadd" OR process.name=="useradd" OR process.name=="groupdel" OR process.name=="groupmod" OR process.name=="userdel" OR process.name=="usermod" OR +contains/message/pam_unix
```

Comment:
We could avoid a 250-character line that is difficult to follow with a slightly simpler notation, something like the following:
```
isAuthProcess: >-
  process.name=="sshd"
  OR process.name=="sudo"
  OR process.name=="groupadd"
  OR process.name=="useradd"
  OR process.name=="groupdel"
  OR process.name=="groupmod"
  OR process.name=="userdel"
  OR process.name=="usermod"
  OR +contains/message/pam_unix
```
What do you think?

Reply:
Great.